package/CHANGELOG

```diff
@@ -13,11 +13,14 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, pgbarletta
+
```

Comment:
```suggestion
```
Please don't add the extra line here (we try to keep the format of this file reasonably strict to make life easier on releases).